apiserver/facades/agent/agent/agent.go

```diff
@@ -34,7 +34,7 @@ type ControllerConfigService interface {
 type AgentAPI struct {
 	*common.PasswordChanger
 	*common.RebootFlagClearer
-	*common.ModelWatcher
+	*common.MongoModelWatcher
```

Comment:
Let's not do this embedding when we cut over to the new one; rather have a member and wrap the calls.

Embedding has made facade versioning difficult in the past.